### ISSUE TITLE
Add an option to toggle notifications on the phone, if browser connected

### ISSUE THREAD

**Issue by RAYs3T** [threema-ch/threema-web#21]
...so that your phone is quiet and won't vibrate/blink on any notifications, you have already seen on the computer.

Maybe check, if the tab in the browser has focus. If this is the case, switch to quite-mode

**Comment by rugk** [comment]
Maybe not an option, but if you've read your message on your desktop (and really read it, …) hide the notification on your mobile phone, if that is possible.

**Comment by Anke** [comment]
While or after having used ThreemaWeb I notice that sometimes I'm not getting any notifications on my phone (and thus missed a meeting yesterday). 
Contrary to others I don't want to miss phone notifications because I'm often leaving my PC or I have several windows open, missing a pop-up notification. Therefore I'd like to get notifications on both devices. (I don't quite understand how Threema would be able to know if messages have already been read, if I leave the PC with an open chat.) Surely an option for individual settings would be ideal - if possible.

**Comment by dbrgn** [comment]
@Anke I guess if you're away from your computer, the computer is not unlocked and with the threema window focussed right? In that case, resolution of #17 should help you too. We're waiting for a feature from an external library until the issue can be closed.

**Comment by Anke** [comment]
@dbrgn No, home office and only me, computer not locked and Threema tab sometimes focussed, more often not.

**Comment by RAYs3T** [comment]
@Anke But the browser notification will stay open until you touch/move your mice again. So you won't miss any new notification in that case. Beside that you should be able to turn of this functionality.

**Comment by Anke** [comment]
My browser is secondary. Relyable notifications on my phone are important to me, and sporadically I don't get any after I've been using ThreemaWeb. I couldn't really detemine what might be causing it.

**Comment by dbrgn** [comment]
> I couldn't really detemine what might be causing it.

Threema Web marks the messages as read (thus hiding the notification on the phone) if the conversation is active.

After the fix in #17, it will only mark them as read if the conversation is active *and* if the browser window is open and focussed.

**Comment by Anke** [comment]
I see, thank you.

**Comment by suamikim** [comment]
As suggested in #494 to me it would be ideal to have an explicit setting within the app which allows us to mute all notification sounds & vibrations temporarily as long as at least 1 web client is connected.

**Comment by lgrahl** [comment]
@suamikim Since it's an option anyway, this could be made configurable easily so everyone is happy.
